.history/README_20200224200626.md

```diff
@@ -0,0 +1,34 @@
+# AWS CLI Orb [![CircleCI status](https://circleci.com/gh/CircleCI-Public/aws-cli-orb.svg "CircleCI status")](https://circleci.com/gh/CircleCI-Public/aws-cli-orb) [![CircleCI Orb Version](https://img.shields.io/badge/endpoint.svg?url=https://badges.circleci.io/orb/circleci/aws-cli)](https://circleci.com/orbs/registry/orb/circleci/aws-cli) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/CircleCI-Public/aws-cli-orb/master/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
```

Comment:
Could you exclude the files in .history from the PR?

Reply:
excluded the files